### ISSUE TITLE
`test_rechunk_striping` anomaly

### ISSUE THREAD

**Issue by fjetter** [coiled/coiled-runtime#755]
@hendrikmakait any idea why this was improving and is now regressing again?

![image](https://user-images.githubusercontent.com/8629629/229106935-405e5390-641e-4967-a32b-112d0f9e80f8.png)

**Comment by fjetter** [comment]
Same is happening elsewhere

![image](https://user-images.githubusercontent.com/8629629/229107610-dd5a599d-9b91-4cad-9dd5-385e57f6d683.png)

![image](https://user-images.githubusercontent.com/8629629/229107687-068aa790-367c-4e70-be5d-b729bd8d7a17.png)

**Comment by ntabris** [comment]
It might be platform change to disk that was made and reverted. I'll check disk rates and see if that's it...

**Comment by dchudz** [comment]
Just curious, does `test_rechunk_striping` intend to use the disk?

I can tell from the name that `test_spilling` does, so this isn't shocking for that one.

**Comment by fjetter** [comment]
> Just curious, does test_rechunk_striping intend to use the disk?

Yes, everything with the "p2p" tag is guaranteed to use disk. This will read and write the entire data once to disk.
